test/scala/com/github/johnynek/bazel_deps/ParseTest.scala

```diff
@@ -106,6 +107,7 @@ class ParseTest extends FunSuite {
               None,
               None,
               None,
+              None,
```

Comment:
can you add some examples that do have licenses?

Reply:
Added to "parse a file with options, yaml"